src/js/entities-service/entities/actions.js

```diff
@@ -66,45 +66,36 @@ const _Model = BaseModel.extend({
       },
     });
   },
-  saveClinician(clinician) {
-    return this.save({ _clinician: clinician.id, _role: null }, {
-      relationships: {
-        clinician: this.toRelation(clinician),
-      },
-    });
-  },
-  saveRole(role) {
-    return this.save({ _role: role.id, _clinician: null }, {
+  saveOwner(owner) {
+    return this.save({
+      _owner: owner,
+    },
+    {
       relationships: {
-        role: this.toRelation(role),
+        owner: owner ? this.toRelation(owner) : null,
```

Comment:
This should be able to be `owner: this.toRelation(owner)` if `owner` is `null` this will set owner to `{ data: null }` which is correct `owner: null` is invalid

Reply:
Oh, this explains the `expect(data.relationships.owner).to.be.null;` above. Thanks.